onnx/defs/experiments/defs.cc

```diff
@@ -106,28 +106,28 @@ NOTE: Currently, it supports data type of float, int32, int64, and bool.
     .Attr(
         "value",
         "The value for the elements of the output tensor.",
-        AttributeProto::FLOAT)
+        0.0f)
     .Attr(
         "dtype",
         "The data type for the elements of the output tensor."
         "Strictly must be one of the types from DataType enum in TensorProto.",
-        AttributeProto::INT)
+        static_cast<int64_t>(1))
```

Comment:
This is not clear, we cannot easily figure out what is the element type.

Reply:
changed to static_cast<int64_t>(TensorProto::FLOAT), which should be good enough.